pkg/api/v1/resource/helpers.go

```diff
@@ -285,7 +285,7 @@ func MergeContainerResourceLimits(container *v1.Container,
 	if container.Resources.Limits == nil {
 		container.Resources.Limits = make(v1.ResourceList)
 	}
-	// NOTE: we exclude hugepages-* resources because hugepages are never overcommitted.
+	// NOTE: we exclude hugepages-* resources because hugepages are never over-committed.
```

Comment:
revert this, "overcommit" is used thoughout the codebase, so I don't think this is worth changing

Reply:
okay